src/txmempool.h

```diff
@@ -338,7 +338,7 @@ class CTxMemPool
      * check does nothing.
      */
     void check(const CCoinsViewCache *pcoins) const;
-    void setSanityCheck(bool _fSanityCheck) { fSanityCheck = _fSanityCheck; }
+    void setSanityCheck(double dFrequency = 1.0) { nCheckFrequency = dFrequency * 4294967296.0; }
```

Comment:
Not sure: but would't it be more clean to use `ULONG_MAX` here? Also not sure: if using `4294967296` instead of `4294967295`(= ULONG_MAX) wouldn't this result in autocast to `uint64_t`?